src/main/java/com/gtnewhorizons/gravisuiteneo/mixins/MixinItemVajra.java

```diff
@@ -52,4 +52,51 @@ public int getHarvestLevel(ItemStack stack, String toolClass) {
             @SuppressWarnings("rawtypes") List tooltip, boolean advancedTooltips, CallbackInfo ci) {
         tooltip.add(EnumChatFormatting.AQUA + StatCollector.translateToLocal("message.vajra.clickRightForSilk"));
     }
+
+    // The vajra onItemUse has two branches for figuring out what a block drops
+    // depending on whether or not the block canSilkHarvest. Then the branches come back together and set the block
+    // to air.
+    // the fixCallOrder redirect handles everything, so there is no need for these calls to do anything.
+    @Redirect(
+            method = "onItemUse",
+            at = @At(
+                    value = "INVOKE",
+                    ordinal = 0,
+                    target = "Lnet/minecraft/block/Block;onBlockHarvested(Lnet/minecraft/world/World;IIIILnet/minecraft/entity/player/EntityPlayer;)V"))
+    private void onBlockHarvestToNoOp(Block block, World world, int x, int y, int z, int meta, EntityPlayer player) {
```

Comment:
Nit: prefix the method names with `gravisuiteneo$` (same with the three methods below).